### PR TITLE
allow @WithPlugin to install multiple plugins

### PR DESCRIPTION
**Pull request by daspilker**

### CODE REVIEW

**Comment by KostyaSha** [comment]
Btw, do you know analogue for `@WithoutPlugin`?

File: test/src/main/java/org/jvnet/hudson/test/recipes/WithPlugin.java [hunk 1]
```diff
@@ -50,9 +50,9 @@
 @Retention(RUNTIME)
 public @interface WithPlugin {
     /**
-     * Name of the plugin.
+     * Whitespace separated list of plugin names.
```
Comment (KostyaSha) [inline test/src/main/java/org/jvnet/hudson/test/recipes/WithPlugin.java]: imho whitespace is bad delimiter, i already had experience with AD where whitespace is valid. Now i use only CSV, wouldn't it better to use CSV?

Comment (KostyaSha) [inline test/src/main/java/org/jvnet/hudson/test/recipes/WithPlugin.java]: Why not array or list like usually annotations accepting?

Comment (daspilker) [inline test/src/main/java/org/jvnet/hudson/test/recipes/WithPlugin.java]: OK, I changed the regex to parse a CSV value.

I thought about using an array. But for compatibility, the non-array value must be kept. And if there are both, array and non-array, which should take precedence? Isn't it confusing to have both?

**Comment by jtnord** [comment]
Im with @KostyaSha on this take an array. There should not be backward compatability issues, as for a single value you should not need the array inittializer - alla `@SupressWarnings`

**Comment by jtnord** [comment]
@jglick do you know how the plugin compat tester will be affected by my sugestion?

**Comment by daspilker** [comment]
OK, I changed this to use an array.

**Comment by oleg-nenashev** [comment]
LGTM :+1:

**Pull request merged**